### PR TITLE
[REST API] Better description for order's status collection param

### PR DESCRIPTION
**Pull request by claudiosanches**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Just a small wording change to reflect the fact that it's possible filter by multiple statuses.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by claudiosanches** [comment]
@peterfabian just need to sort this string first, then we can update on docs too.

File: includes/api/class-wc-rest-orders-controller.php [hunk 1]
```diff
@@ -257,7 +257,7 @@ public function get_collection_params() {
 
 		$params['status'] = array(
 			'default'           => 'any',
-			'description'       => __( 'Limit result set to orders assigned a specific status.', 'woocommerce' ),
+			'description'       => __( 'Limit result set to orders assigned to specific statuses.', 'woocommerce' ),
```
Comment (peterfabian) [inline includes/api/class-wc-rest-orders-controller.php]: I think the wording is a bit unusual, orders are not assigned to statuses. I would use sth like 'Limit result set to orders assigned one of the specified statuses.' or 'Limit result set to orders having specific statuses assigned'?
@mikejolley can you please help as a native speaker?

Comment (mikejolley) [reply includes/api/class-wc-rest-orders-controller.php]: ..well it's not bad just a bit of a mouthful. I'd go with:

```
List orders which have specific statuses.
```

Comment (claudiosanches) [reply includes/api/class-wc-rest-orders-controller.php]: @mikejolley `Limit result set to` is like a standard on the REST API: https://developer.wordpress.org/rest-api/reference/posts/#arguments

Comment (peterfabian) [reply includes/api/class-wc-rest-orders-controller.php]: Let's go with `Limit result set to orders which have specific statuses` then, right?

**Review by peterfabian — APPROVED** [review]
👍

**Pull request merged**